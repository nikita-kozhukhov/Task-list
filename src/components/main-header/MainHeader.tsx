import styles from './MainHeader.module.scss';

export const MainHeader = () => {
  return <h1 className={styles['main-header']}>Создать задачу</h1>;
};
