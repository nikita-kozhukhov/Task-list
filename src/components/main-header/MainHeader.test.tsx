import { render, screen } from '@testing-library/react';
import { MainHeader } from './MainHeader';

describe('MainHeader', () => {
  test('отображает корректный заголовок', () => {
    render(<MainHeader />);
    expect(screen.getByRole('heading', { level: 1 })).toHaveTextContent(
      'Создать задачу'
    );
  });

  test('совпадает с предыдущей версией снапшота', () => {
    const { container } = render(<MainHeader />);
    expect(container).toMatchSnapshot();
  });
});
