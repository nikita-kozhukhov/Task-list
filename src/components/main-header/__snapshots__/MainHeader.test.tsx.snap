// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing

exports[`MainHeader совпадает с предыдущей версией снапшота 1`] = `
<div>
  <h1
    class="main-header"
  >
    Создать задачу
  </h1>
</div>
`;
